tests/test_operations.py

```diff
@@ -1509,5 +1511,50 @@ def test_invalid_secrets(self):
             s.run()
 
 
+@testmode.integration
+class TestFlacTableOperations(unittest.TestCase):
+    file_keys = [f'files/{uuid.uuid4()}.{datetime_to_version_format(datetime.datetime.now())}' for x in range(4)]
+    bundle_keys = [f'bundles/{uuid.uuid4()}.{datetime_to_version_format(datetime.datetime.now())}' for x in range(1)]
+    all_keys = file_keys + bundle_keys
+    groups = ["testing", "operations"]
+
+    def test_flac_flow(self):
+        self._test_upload_keys()
+        self._test_modify_key()
```

Comment:
Missing a test for Get https://github.com/DataBiosphere/data-store/pull/144/files#diff-8939c1d43d7bda0a15f6469ce7d7bc94R42

Reply:
remove/delete-ing objects use `get` functionality, but ill add another one :)

Reply:
@chmreid i added an explicit test for GET functionality